projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_category-chart.scss

```diff
@@ -11,8 +11,8 @@
 /// @property {List} marker-brushes [series]- Defines the palette from which automatically assigned series marker brushes colors are selected.
 /// @property {List} negative-brushes [series] - Defines the palette used for coloring negative items in a chart type with contextual coloring, such as Waterfall.
 /// @property {List} outlines [series] - Defines the palette from which automatically assigned series outline colors are selected.
-/// @property {List} marker-outlines [series] - Defines the palette from which automatically assigned series marker outline colors are selected.
-/// @property {List} negative-outlines [series] - Defines the brushes used for drawing negative elements in a chart type with contextual coloring, such as Waterfall.
+/// @property {Color} marker-outlines [hsl(347, 82%, 64%)] - Defines the palette from which automatically assigned series marker outline colors are selected.
+/// @property {Color} negative-outlines [hsl(347, 82%, 64%)] - Defines the brushes used for drawing negative elements in a chart type with contextual coloring, such as Waterfall.
```

Comment:
Also not sure this affects your SASS docs generation, but if it does - negative brushes/outlines (as the names suggest) are still lists. We're just setting a singular value in them by default, right?